cypress/e2e/news/createReadDeleteNewsInAdvance.feature

```diff
@@ -0,0 +1,49 @@
+@stable_test
```

Comment:
feature file name should include term school -> createReadDeleteSchoolNewsInAdvance.feature